src/platform/elements/date-picker/DatePickerInput.ts

```diff
@@ -45,20 +46,28 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
   @Input() placeholder: string;
   @Input() maskOptions: any;
   @Input() format: string;
+  @Input() specialFormat: boolean;
```

Comment:
All the code looks solid. Not loving the name `specialFormat`. Can you think of a better name that is more descriptive?